src/amchart/CommonSerial.js

```diff
@@ -252,6 +252,9 @@
             dataObjArr.sort(function (a, b) {
                 return a[sortField] - b[sortField];
             });
+            this.data().sort(function (a, b) {
```

Comment:
i think if we just sort the data before its used at the top of the function we can eliminate sorting twice?